packages/web/src/components/form2/Field.tsx

```diff
@@ -119,6 +119,12 @@ export default function Field({
     }
   }, [rules]);
 
+  useEffect(() => {
+    if (inlineEdit && field?.fieldType === 'response') {
+      setAddOption({ ...addOption, showDrawer: true });
+    }
```

Comment:
From requirement standpoint what I see is
user is creating response - so whatever is the code implementations for this requirement should be in response component? if not why? when fieldType = response then get the formId and display the form or will be this in the form view related component? @saichidvi

Reply:
Feedback - I am not asking u to
Explain one line of code - u should know by now I understand lines or code. Like I was always asking u since the first day of hiring you - what is the component and props and the requirement given is it implemented in the correct component , correct prop is used and correct t hooks are used @saichidvi - so the implementation I am asking abt why this component is changed why this prop is changed and why this hook is changed for this requirement

Reply:
> If you see , we will use the formView to fill the form . But at that time you did not opposed like this (you were filling the form and creating the response then it should be in response related component ? ).
> The implementation should be present in the field component itself madam.

@saichidvi

Reply:
Who and when and why showDrawer is set to true ?

Reply:
> Who and when and why showDrawer is set to true ?

@saichidvi

Reply:
> 

@saichidvi not sure what u mean by at that time u didn’t opposed ? 
the reason for
Bringing people to work on this project version 0.2 is to look at components and props And relationships and implement correctly. 
Initially I brought Vivek so u can ask him questions daily and learn from him - and now I am reviewing the coding ? Always react principles have to be applied whether I review code or not review code.

Reply:
> From requirement standpoint what I see is user is creating response - so whatever is the code implementations for this requirement should be in response component? if not why? when fieldType = response then get the formId and display the form or will be this in the form view related component? @saichidvi

If you see we have many functionalities in this component (formView) 
https://github.com/Codemarket2/boossti-frontend/blob/saichidvi3/packages/web/src/components/form2/FormView.tsx#L505
which are must needed while user is filling the form .. so , all these functionalities ,validations and checks are needed while creating the formResponse , so we use this formView for this reason.